sr_grip_feedback/package.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0"?>
+<package>
+  <name>sr_grip_feedback</name>
+  <version>0.0.0</version>
+  <description>The sr_grip_feedback package</description>
+
+  <!-- One maintainer tag required, multiple allowed, one person per tag --> 
+  <!-- Example:  -->
+  <!-- <maintainer email="jane.doe@example.com">Jane Doe</maintainer> -->
+  <maintainer email="ethan@shadowrobot.com">Ethan Fowler</maintainer>
```

Comment:
Please use the same maintainer we have in recent packages. (e.g. https://github.com/shadow-robot/sr_interface/blob/indigo-devel/sr_robot_commander/package.xml#L6).

You can add yourself as author.